ui/perfherder/graphs/GraphsContainer.jsx

```diff
@@ -241,14 +242,15 @@ class GraphsContainer extends React.Component {
   // padding for the y axis tick labels, so this is a workaround (setting state
   // doesn't work with this callback, which is why a class property is used instead)
   setLeftPadding = (tick, index, ticks) => {
-    const highestTickLength = ticks[ticks.length - 1].toString();
+    const formattedNumber = numeral(tick).format('0.0a');
```

Comment:
Could we extract this custom formatting functionality into a separate function/method?
Because we seem to duplicate it in 2 consecutive lines.